bluepyemodel/emodel_pipeline/emodel_pipeline.py

```diff
@@ -316,6 +319,26 @@ def summarize(self):
 
         print(self.access_point)
 
+    def check_name(self, emodel_name):
+        """Check if name complies with requirements:
+        https://nrn.readthedocs.io/en/8.2.3/guide/hoc_chapter_11_old_reference.html#names"""
+
+        allowed_chars = string.ascii_letters + string.digits + "_"
+        translate_args = [str.maketrans("", "", allowed_chars)]
```

Comment:
I think that you don't have to put this in a list, and don't have to use `*` in translate later